lib/matplotlib/offsetbox.py

```diff
@@ -946,8 +946,12 @@ def __init__(self, loc, *,
             See the parameter *loc* of `.Legend` for details.
         pad : float, default: 0.4
             Padding around the child as fraction of the fontsize.
-        borderpad : float, default: 0.5
+        borderpad : float or (float, float), default: 0.5
             Padding between the offsetbox frame and the *bbox_to_anchor*.
+            If a float, the same padding is used for both x and y.
+            If a tuple of two floats, it specifies the (x, y) padding.
+        .. versionadded:: 3.11
```

Comment:
This need a blank line above it so it renders correctly.  Compare:
[AnchoredOffsetBox](https://output.circle-artifacts.com/output/job/ad8c58b8-d68e-4997-9980-4eb75209c84a/artifacts/0/doc/build/html/api/offsetbox_api.html#matplotlib.offsetbox.AnchoredOffsetbox)
[inset_axes](https://output.circle-artifacts.com/output/job/ad8c58b8-d68e-4997-9980-4eb75209c84a/artifacts/0/doc/build/html/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html#mpl_toolkits.axes_grid1.inset_locator.inset_axes) (correct)